components-list.html

```diff
@@ -1,160 +1,156 @@
-            style="width: 144px; height: 124px"
-            class="component-card-img"
-          />
-        </div>
-      </a>
-      <a href="components/input/input.html" class="component-card">
-        <div class="component-name">
-          <span>Inputs</span>
-        </div>
-        <div class="img-container">
-          <img
-            src="/assets/inputs.svg"
-            style="width: 160px; height: 115px"
-            class="component-card-img"
-          />
-        </div>
-      </a>
-      <a href="components/navbar/navbar.html" class="component-card">
-        <div class="component-name">
-          <span>Navigation</span>
-        </div>
-        <div class="img-container">
-          <img
-            src="/assets/navbar.svg"
-            style="width: 224px; height: 78px"
-            class="component-card-img"
-          />
-        </div>
-      </a>
-      <a href="components/rating/rating.html" class="component-card">
-        <div class="component-name">
-          <span>Rating</span>
-        </div>
-        <div class="img-container">
-          <img
-            src="/assets/rating.svg"
-            style="width: 224px; height: 115px"
-            class="component-card-img"
-          />
-        </div>
-      </a>
-      <a href="components/typography/typography.html" class="component-card">
-        <div class="component-name">
-          <span>Typography</span>
-        </div>
-        <div class="img-container">
-          <img
-            src="/assets/typography.svg"
-            style="width: 224px; height: 115px"
-            class="component-card-img"
-          />
-        </div>
-      </a>
-    </main>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/css/components-list.css" />
+  <link rel="icon" type="image/x-icon" href="/assets/favicon.ico" />
+  <title>Components | Raptor UI</title>
+</head>
+
+<body>
+  <header class="header-nav">
+    <a href="index.html" class="header-logo-link">
+      <img class="header-logo-img" src="/assets/jurassic-park-logo.png" alt="jurassic-park-logo" />
+      <p class="header-logo-name">Raptor UI</p>
+    </a>
+    <div class="header-nav-links">
+      <nav>
+        <ul>
+          <li><a href="docs.html" class="header-link">Docs</a></li>
+          <li><a href="https://github.com/MarkVed17/raptor-ui" class="header-link">Github</a></li>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+
+  <p class="page-title">Components</p>
+
+  <!-- Alerts -->
+  <main class="main-content">
+    <a href="components/alert/alert.html" class="component-card">
+      <div class="component-name">
+        <span>Alerts</span>
+      </div>
+      <div class="img-container">
+        <img src="/assets/alerts.svg" style="width: 224px; height: 96px" class="component-card-img" />
```

Comment:
remove inline CSS

Reply:
Yes, you’re right. But in this case I’ll have to add separate classes for every component to just alter the dimensions of images which isn’t efficient imo.